packages/salesforce-adapter/src/change_validators/flows.ts

```diff
@@ -96,14 +98,46 @@ const testCoveragePostDeploy = (instance: InstanceElement): DeployActions => ({
   },
 })
 
-const deployAsInactivePostDeploy = (instance: InstanceElement): DeployActions => ({
-  postAction: {
-    title: 'Deploying as inactive',
-    subActions: [
-      `Your salesforce is configured not to allow deployment of active flows, please make sure to manually activate the flow. Flow name: ${instance.elemID.getFullName()}`,
-    ],
-  },
-})
+const deployAsInactivePostDeploy = (instance: InstanceElement, baseUrl: URL | undefined): DeployActions => {
+  const url = _.get(instance, ['annotations', '_service_url'])
```

Comment:
I think this is the proper way of doing this, using the constants 
```suggestion
  const url = instance.annotations[CORE_ANNOTATIONS.SERVICE_URL]
```